textdb/textdb-dataflow/src/main/java/com/google/re2j/RE2J_LICENSE.txt

```diff
@@ -0,0 +1,32 @@
+This is a work derived from Russ Cox's RE2 in Go, whose license
```

Comment:
Just want to confirm: this is BSD license?  If so, we are required to include this license?